### ISSUE TITLE
show all images

### ISSUE THREAD

**Issue by learningchannel** [datenstrom/yellow#147]
hello, i am learning how to create a easy photo log system.
entries are  

```
---
title: melody cake day
published: 2015-11-11
tags: food, drinks, cake
---
take a look!
[gallery 2015-11-11-party-event-1/.]
```

and another example:

```
---
title: new years party
published: 2016-01-02
tag: food, drinks
---
take a look!
[gallery 2016-01-02-party-event-2/.]
```

my media: 
images for the events are in subfolders like this from photo workflow.

```
media/images/YYYY-DD-MM-TITLE/
```
this work flow has been great and smooth so far.

However i'm dreaming to make one "archive" page with all my images from all the subfolders (/images/*) as thumbs. 

Visual example: a tumblr archive: http://hottemper.tumblr.com/archive 

Gallery plugin does a great job with thumbs for one folder with a bunch of images. 
```
[gallery subfolder/. - 10%]
```

is making an archive page possible? -- how about with gallery plugin? 

thank you!

**Comment by BsNoSi** [comment]
Did you try something like

~~~
[gallery 20160115/.]

[gallery 20160114/.]

[gallery 20160113/.]

…

~~~

Each gallery shows a folder, more than one should work on a single page but I did not test that.

**Comment by markseu** [comment]
First example, you can use the [gallery plugin](https://github.com/datenstrom/yellow-extensions/tree/master/plugins/gallery) to show all images in your `media` folder: 

`[gallery . - 64]`

Second example, you can create a page with a gallery archive and add individual galleries:
~~~
---
Title: Gallery archive
---
## 2016-02

[gallery 2016-02-02-party-event-2/. - 64]

## 2015-11

[gallery 2015-11-11-party-event-1/. - 64]
~~~
 
Here's how this could look like:

![screenshot ](https://cloud.githubusercontent.com/assets/2429165/12562479/b3666e7c-c3a5-11e5-8319-ef39cdab3a3b.png)

Hope this helps. :smile:

**Comment by JanYellow** [comment]
In my image directory are pictures after the pattern

name-number.jpg

But the gallery probably can not print the pattern

*-0.jpg

create? Anyway, I did not manage it.

So only show all pictures which end with -0.jpg.

**Comment by JanYellow** [comment]
In the meantime I have simply renamed the files and can thus display the data.

`[gallery folder/0* - 120]`

That is only a stopgap solution for me. In the gallery are animal pictures, example:

abby-0.jpg
abby-1.jpg
abby-2.jpg
...
leon-0.jpg
leon-1.jpg
...
leon-22.jpg
...

abby-0.jpg I use for example in the blog

`[image folder/abby-0.jpg "Dog Abby" left 200 100]`

and

`[gallery folder/abby* simple 400]`

If I want to show a gallery of all dogname-0.jpg on another page, that does not work. Why I rebuilt the file names, 0-dogname.jpg. However I have outsourced these pictures into another directory ;-) I would like to maintain my previous sorting of the picture file!

But maybe I am also obtuse =8-) I am still experimenting with Yellow ... and meanwhile I have become friends with the edit function <g>
